tests/integration/frameworks/test_pytorch_impl.py

```diff
@@ -7,87 +7,42 @@
 import torch.nn as nn
 
 import bentoml
-from bentoml.pytorch import PytorchTensorContainer
 from tests.utils.helpers import assert_have_file_extension
+from bentoml._internal.frameworks.pytorch import PyTorchTensorContainer
 from tests.utils.frameworks.pytorch_utils import test_df
+from tests.utils.frameworks.pytorch_utils import predict_df
 from tests.utils.frameworks.pytorch_utils import LinearModel
-
-
-def predict_df(model: nn.Module, df: pd.DataFrame):
-    input_data = df.to_numpy().astype(np.float32)
-    input_tensor = torch.from_numpy(input_data)
-    return model(input_tensor).unsqueeze(dim=0).item()
-
-
-class LinearModelWithBatchAxis(nn.Module):
-    def __init__(self):
-        super(LinearModelWithBatchAxis, self).__init__()
-        self.linear = nn.Linear(5, 1, bias=False)
-        torch.nn.init.ones_(self.linear.weight)
-
-    def forward(self, x, batch_axis=0):
-        if batch_axis == 1:
-            x = x.permute([1, 0])
-        res = self.linear(x)
-        if batch_axis == 1:
-            res = res.permute([0, 1])
-
-        return res
-
-
-class ExtendedModel(nn.Module):
-    def __init__(self, D_in, H, D_out):
-        """
-        In the constructor we instantiate two nn.Linear modules and assign them as
-        member variables.
-        """
-        super(ExtendedModel, self).__init__()
-        self.linear1 = nn.Linear(D_in, H)
-        self.linear2 = nn.Linear(H, D_out)
-
-    def forward(self, x, bias=0.0):
-        """
-        In the forward function we accept a Tensor of input data and an optional bias
-        """
-        h_relu = self.linear1(x).clamp(min=0)
-        y_pred = self.linear2(h_relu)
-        return y_pred + bias
+from tests.utils.frameworks.pytorch_utils import ExtendedModel
+from tests.utils.frameworks.pytorch_utils import LinearModelWithBatchAxis
 
 
 @pytest.fixture(scope="module")
 def models():
-    def _(test_type):
-        _model: nn.Module = LinearModel()
-        if "trace" in test_type:
-            tracing_inp = torch.ones(5)
-            model = torch.jit.trace(_model, tracing_inp)
-        elif "script" in test_type:
-            model = torch.jit.script(_model)
-        else:
-            model = _model
+    def _():
+        model: nn.Module = LinearModel()
         tag = bentoml.pytorch.save("pytorch_test", model)
         return tag
 
     return _
 
 
-@pytest.mark.parametrize("test_type", ["", "tracedmodel", "scriptedmodel"])
-def test_pytorch_save_load(test_type, models):
-    tag = models(test_type)
-    assert_have_file_extension(bentoml.models.get(tag).path, ".pt")
+def test_pytorch_save_load(models):
+    tag = models()
+    bentoml_model = bentoml.models.get(tag)
+    assert_have_file_extension(bentoml_model.path, ".pt")
+    assert bentoml_model.info.context.get("model_format") == "torch.save:v1"
 
     pytorch_loaded: nn.Module = bentoml.pytorch.load(tag)
     assert predict_df(pytorch_loaded, test_df) == 5.0
 
 
 @pytest.mark.gpus
```

Comment:
actually this mark will only run when we pass `--gpus` to pytest
```bash
pytest -vvv ... --gpus
```
so your cpu test would not be tested here